base/pgsql/pgsql.Deployment.yaml

```diff
@@ -70,8 +70,8 @@ spec:
             cpu: 10m
             memory: 50Mi
         securityContext:
-          runAsUser: 100
-          runAsGroup: 100
+          runAsUser: 999
```

Comment:
Was the user here wrong before? Just double-checking this wasn't a typo

Reply:
wasn't a typo. i checked the user "postgres" and it's 999.  it works with other users too according to the documentation. not sure what's better, but both work. i  did see this in the pgsql logs

```
Warning: you don't own a lock of type ExclusiveLock
```

but i saw this before too with user 100.